test/examples/change_streams.js

```diff
@@ -101,9 +101,9 @@ describe('examples(change-stream):', function() {
       const collection = db.collection('inventory');
       const changeStream = collection.watch();
 
-      let resumeToken, newChangeStream;
+      let newChangeStream;
       changeStream.on('change', next => {
-        resumeToken = next._id;
+        const resumeAfter = next._id;
         changeStream.close();
 
         newChangeStream = collection.watch({ resumeAfter });
```

Comment:
Does this not need to be `"resumeAfter" : resumeToken`? The node docs don't indicate that passing in a resume token alone would work, but that doesn't mean it's not valid (docs could be out of date).

Reply:
In Node >=4, you can use object literal shorthand. So, with any object, `{resumeAfter}` is the same as `{resumeAfter: resumeAfter}`. This is the format that `Changestream Example 3 Alternative` uses right below it.

That said, I would be on board with @kvwalker's [suggestion](https://github.com/mongodb/node-mongodb-native/pull/1918#pullrequestreview-201179321) to do `{resumeAfter: resumeToken}` for clarity purposes.

Reply:
That is super neat and I did not know that. No other comments on my end.